### PR TITLE
Add damping factor

### PR DESCRIPTION
**Pull request by coltc50**
Add damping factor to pv tab. Quick and dirty...

### CODE REVIEW

File: admin/jsonConfig.json [hunk 1]
```diff
@@ -306,6 +306,52 @@
...
+                                "es": "humedeciendo por la mañana",
+                                "pl": "nawilżanie rano",
+                                "uk": "зволоження вранці",
+                                "zh-cn": "早上受潮"
+                            },
+                            "filter": false,
+                            "sort": false,
+                            "default": 0
+                        },
+                                        {
```
Comment (klein0r) [inline admin/jsonConfig.json:333]: Please fix indentation

File: main.js [hunk 2]
```diff
@@ -659,10 +659,10 @@ class Pvforecast extends utils.Adapter {
 			if (this.config.service === 'forecastsolar') {
 				if (this.hasApiKey) {
 					// https://api.forecast.solar/:apikey/estimate/:lat/:lon/:dec/:az/:kwp
-					url = `https://api.forecast.solar/${this.config.apiKey}/estimate/${this.pvLatitude}/${this.pvLongitude}/${plant.tilt}/${plant.azimuth}/${plant.peakpower}?time=utc`;
+					url = `https://api.forecast.solar/${this.config.apiKey}/estimate/${this.pvLatitude}/${this.pvLongitude}/${plant.tilt}/${plant.azimuth}/${plant.peakpower}?time=utc&damping=${plant.dampingm},${plant.dampinge}`;
```
Comment (klein0r) [inline main.js:662]: The name of the config attributes are `dampe` and `dampm`

File: admin/jsonConfig.json [hunk 3]
```diff
@@ -306,6 +306,52 @@
                             "filter": false,
                             "sort": false,
                             "default": 0
+                        },
+                                {
```
Comment (klein0r) [inline admin/jsonConfig.json:310]: Please fix indentation

**Review by klein0r — COMMENTED** [review]
- Add changelog to README
- Add damping / conversion for other providers (e.g. solcast)
- Update documentation

**Comment by coltc50** [comment]
Sorry, was too late yesterday... now I correct all typos and it should work.

**Comment by klein0r** [comment]
> but `"hidden": "data.service !== 'forecastsolar'",` dosen't work at this time...

In tables you have to use `globalData` - `data` references other fields in the same row.

> don't know what's wrong or what I missing?

Nothing. That's an issue with the schema validation.

**Pull request merged**